dtfs-central-api/src/v1/controllers/portal/facility/get-latest-amendment-facility-value-and-cover-end-date.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { ApiError, CustomExpressRequest } from '@ukef/dtfs2-common';
+import { HttpStatusCode } from 'axios';
+import { Response } from 'express';
+import { TfmFacilitiesRepo } from '../../../../repositories/tfm-facilities-repo';
+
+type GetLatestAmendmentValueAndCoverEndDateByFacilityIdRequestParams = { facilityId: string };
+export type GetLatestAmendmentValueAndCoverEndDateByFacilityIdRequest = CustomExpressRequest<{
+  params: GetLatestAmendmentValueAndCoverEndDateByFacilityIdRequestParams;
+}>;
+
+/**
+ * get latest amendment value and cover end date by facilityId
+ * @param req - request
+ * @param res - response
+ */
+export const getLatestAmendmentFacilityValueAndCoverEndDate = async (req: GetLatestAmendmentValueAndCoverEndDateByFacilityIdRequest, res: Response) => {
+  const { facilityId } = req.params;
+
+  try {
+    const valueAmendment = await TfmFacilitiesRepo.findLatestValueAmendmentByFacilityId(facilityId);
+    const coverEndDateAmendment = await TfmFacilitiesRepo.findLatestCoverEndDateAmendmentByFacilityId(facilityId);
+
```

Comment:
Validate `valueAmendment` and `coverEndDateAmendment` before refering to its value please.